Python/Module4_OOP/Methods.md

```diff
@@ -200,7 +200,7 @@ __main__.Dummy
 `dict.fromkeys` is an example of a class method that takes in an iterable, and returns a dictionary whose keys are the elements of that iterable, and whose values all default to `None`.
 
 ```python
->>> dict.fromkeys("abcd")
+>>> dict.fromkeys("abcd",2.3)
```

Comment:
Add a space here to be [pep-8 compliant](https://www.pythonlikeyoumeanit.com/Module5_OddsAndEnds/Writing_Good_Code.html#The-PEP8-Style-Guide-to-Python-Code)
```suggestion
>>> dict.fromkeys("abcd", 2.3)
```